example/aws/cognito/README.md

```diff
@@ -0,0 +1,270 @@
+            2. `kubernetes.io/role/internal-elb`
+            3. `kubernetes.io/role/elb`
+    2. Create an IAM role to use via service account
+        1. Substitute values for the CLUSTER_NAME and AWS_REGION in the script below
+        2. ```
+            # Replace the CLUSTER_NAME and AWS_REGION
+            export CLUSTER_NAME=<>
+            export AWS_REGION=<>
+            export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query "Account" --output text)
+            
+            eksctl utils associate-iam-oidc-provider --cluster ${CLUSTER_NAME} \
+            --region ${AWS_REGION} --approve
+            
+            export OIDC_PROVIDER_URL=$(aws eks describe-cluster --name $CLUSTER_NAME --region $AWS_REGION --query "cluster.identity.oidc.issuer" --output text | cut -c9-)
+            export IRSA_ROLE_NAME=kf-admin-${AWS_REGION}-${CLUSTER_NAME}
+            
+            printf '{
+                "Version": "2012-10-17",
+                "Statement": [
+                {
+                    "Effect": "Allow",
+                    "Principal": {
+                    "Federated": "arn:aws:iam::'$AWS_ACCOUNT_ID':oidc-provider/'$OIDC_PROVIDER_URL'"
+                    },
+                    "Action": "sts:AssumeRoleWithWebIdentity",
+                    "Condition": {
+                    "StringEquals": {
+                        "'$OIDC_PROVIDER_URL':aud": "sts.amazonaws.com",
+                        "'$OIDC_PROVIDER_URL':sub": [
+                        "system:serviceaccount:kubeflow:fluentd",
+                        "system:serviceaccount:kubeflow:kf-admin",
+                        "system:serviceaccount:kubeflow:alb-ingress-controller",
+                        "system:serviceaccount:kubeflow:profiles-controller-service-account"
+                        ]
+                    }
+                    }
+                }
+                ]
+            }
+            ' > ./trust.json
+            
+            
+            aws iam create-role --role-name $IRSA_ROLE_NAME --assume-role-policy-document file://trust.json
+            aws iam put-role-policy --role-name $IRSA_ROLE_NAME --policy-name iam_alb_ingress_policy --policy-document file://distributions/aws/infra_configs/iam_alb_ingress_policy.json
+            aws iam put-role-policy --role-name $IRSA_ROLE_NAME --policy-name iam_profile_controller_policy --policy-document file://distributions/aws/infra_configs/iam_profile_controller_policy.json
+            
+            export IAM_ROLE_ARN_FOR_IRSA=$(aws iam get-role --role-name $IRSA_ROLE_NAME --output text --query 'Role.Arn')
+            ```
+    3. Substitute values for ALB deployment
+        1. ```
+            printf 'clusterName='$CLUSTER_NAME'' > distributions/aws/aws-alb-ingress-controller/base/params.env
+            ```
+5. Deploy Kubeflow
```

Comment:
is it possible to replace this with one command ? like build from examples folder?

Reply:
TODO planned for next PR